nupic/research/frameworks/pytorch/model_utils.py

```diff
@@ -274,3 +298,41 @@ def deserialize_state_dict(fileobj, device=None):
         # FIXME: Backward compatibility with old uncompressed checkpoints
         state_dict = torch.load(fileobj, map_location=device)
     return state_dict
+
+
+def soft_cross_entropy(output, target, size_average=True):
+    """ Cross entropy that accepts soft targets
+    Args:
+         output: predictions for neural network
+         targets: targets, can be soft
+         size_average: if false, sum is returned instead of mean
```

Comment:
Avoid mixed docstring styles on the same file. To be consistent the docstring should be:
```
         :param output: predictions for neural network
         :param targets: targets, can be soft
         :param size_average: if false, sum is returned instead of mean
```